content/tutorials/portenta-h7/por-ard-gs/content.md

```diff
@@ -85,7 +85,9 @@ Now it's time to upload the sketch and see if the LED will start to blink. Make
 
 ![Selecting the Arduino Portenta H7 (M7 core)](assets/por_gs_board_selection_pro_ide.png)
 
-# Conclusion
+**Optional:** We collect all the sketches from the tutorials in a library wrapper, you can find it at: *Tools -> Library Manager -> Arduino_Pro_Tutorials* or [download them from the repository](https://github.com/arduino-libraries/Arduino_Pro_Tutorials/releases).
```

Comment:
```suggestion
**Optional:** We collect all the sketches from the tutorials in a library which you can install from the Library Manager: **Tools -> Manage Libraries**. Search for 'Arduino_Pro_Tutorials' or download them from the [repository](https://github.com/arduino-libraries/Arduino_Pro_Tutorials/releases).
```

Reply:
@marqdevx The tools menu doesn't have a 'Library Manager', at least not on macOS. Is this different on other OS'?

Reply:
My mistake, you push that button and then it opens the library manager, my brain shortcirtuit there hehe.
Thanks!